Core/DefInjected/TipSetDef/Tips.xml

```diff
@@ -1,137 +1,137 @@
   <!-- EN: Turrets explode when they take a lot of damage. Don't put them too close together, and don't put your people too close to them. -->
-  <GameplayTips.TurretsExplode>TODO</GameplayTips.TurretsExplode>
+  <GameplayTips.TurretsExplode>Le torrette esplodono quando subiscono molti danni. Non posizionarle troppo vicine l'una all'altra e tieni i coloni a debita distanza.</GameplayTips.TurretsExplode>
   <!-- EN: Some animals explode upon death. You can use transport pods to drop animals on enemies. Think about it. -->
-  <GameplayTips.AnimalDeathExplosion>TODO</GameplayTips.AnimalDeathExplosion>
+  <GameplayTips.AnimalDeathExplosion>Alcuni animali esplodono appena muoiono. Puoi sfruttare questo a tuo vantaggio, usando le capsule di trasporto per lanciare gli animali sui nemici.</GameplayTips.AnimalDeathExplosion>
   <!-- EN: If you need help in a fight, call your allies using the comms console. -->
-  <GameplayTips.CallAllies>TODO</GameplayTips.CallAllies>
+  <GameplayTips.CallAllies>Se necessiti di aiuto durante uno scontro, chiama gli alleati tramite la console delle comunicazioni.</GameplayTips.CallAllies>
   <!-- EN: Maddened animals will attack any human, including your enemies. You can use this. -->
-  <GameplayTips.MaddenedAnimals>TODO</GameplayTips.MaddenedAnimals>
+  <GameplayTips.MaddenedAnimals>Gli animali impazziti attaccheranno chiunque, anche i tuoi nemici. Usa questo a tuo vantaggio.</GameplayTips.MaddenedAnimals>
   <!-- EN: EMP bursts will temporarily disable turrets and shields. -->
-  <GameplayTips.EMPDisablesTurretsShields>TODO</GameplayTips.EMPDisablesTurretsShields>
+  <GameplayTips.EMPDisablesTurretsShields>Le scariche EMP disabilitano temporaneamente torrette e scudi.</GameplayTips.EMPDisablesTurretsShields>
   <!-- EN: EMP bursts instantly break personal shields. -->
-  <GameplayTips.EMPDisablesPersonalShields>TODO</GameplayTips.EMPDisablesPersonalShields>
+  <GameplayTips.EMPDisablesPersonalShields>Le scariche EMP disabilitano istantaneamente gli scudi portatili.</GameplayTips.EMPDisablesPersonalShields>
   <!-- EN: The hunter stealth stat reduces the chance of animal attacks. It is affected by the hunter's animals and shooting skills. -->
-  <GameplayTips.HunterStealth>TODO</GameplayTips.HunterStealth>
+  <GameplayTips.HunterStealth>L'abilità cacciatore silenzioso, riduce la possibilità che un animale reagisca. Essa dipende dalle abilità Animali e Tiro.</GameplayTips.HunterStealth>
   <!-- EN: Animals are more likely to attack when harmed from close range. Long-range, slow-firing weapons are safest for hunting. -->
-  <GameplayTips.AnimalsAttackCloseRange>TODO</GameplayTips.AnimalsAttackCloseRange>
+  <GameplayTips.AnimalsAttackCloseRange>E' molto probabile che un animale reagisca ad un attacco corpo a corpo. Le armi a distanza, con bassa cadenza di tiro, sono le più sicure per la caccia.</GameplayTips.AnimalsAttackCloseRange>

```

Comment:
Qui puoi usare " È " al posto di " E' "

Reply:
Se posso lasciare un suggerimento, eviterei il "attacco corpo a corpo" perché non esprime appieno il concetto, in quanto anche gli attacchi con armi a distanza, però effettuati vicino all'animale, possono causare la sua reazione. Forse è meglio un "attacco ravvicinato" che può includere sia il c.a.c. che gli attacchi ravvicinati con armi da fuoco

Reply:
Sì é vero hai ragione, allora sarebbe meglio "attacco ravvicinato" al posto di "attacco corpo a corpo".

Reply:
e se mettessimo "con armi da mischia"?

Reply:
No, perché questo tooltip evidenzia il rischio che un animale reagisca ad un attacco ravvicinato, sia corpo a corpo che con armi a distanza. Se usi il fucile a pompa contro un branco di struzzi, è garantito che cercheranno di farti la pelle

Reply:
corretto